### PR TITLE
feat: add loader for exports from kialo.com

### PR DESCRIPTION
**Pull request by leonkamke**

### CODE REVIEW

File: src/load/kialo.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
+import * as model from "../model/index.js";
+import assert from "assert";
```
Comment (mirkolenz) [inline src/load/kialo.ts:2]: why do we need a new dependency here? i would like to limit the dependencies to the bare minimum if possible

File: src/load/kialo.ts [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            As long as the pattern is not found in the next line,
+            we assume that the text belongs to the previous statement.
+            */
+      while (next_line !== undefined && next_line.match(pattern) === null) {
+        text += "\n" + next_line.trim();
+        next_line = iter.next().value;
+      }
+
+      assert(source_id !== undefined);
+      assert(text !== undefined);
```
Comment (mirkolenz) [inline src/load/kialo.ts:67]: Maybe replace it with if/throw Error?

File: src/test/argdown.test.ts [hunk 3]
```diff
@@ -164,5 +164,5 @@ disposition for criminal behaviour.
 
 test("graph: aif2arguebuf", () => {
   const g = arguebuf.load.argdown(graph);
-  console.log(g);
```
Comment (mirkolenz) [inline src/test/argdown.test.ts:167]: Could you remove this line completely?

File: src/test/kialo.test.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import { assertType, expect, test } from "vitest";
+import * as arguebuf from "../index.js";
+import * as model from "../model/index.js";
+import * as fs from "fs";
+
+const file_path = "src/test/kialoTestFile.txt";
```
Comment (mirkolenz) [inline src/test/kialo.test.ts:6]: did you try to write `import file from "./kialoTestFile.txt"? If I remember correctly, vite should be able to do that. Then you would not need to read the file manually.

**Review by mirkolenz — CHANGES_REQUESTED** [review]
We should not include the kialo file in here as these may cause license issues. I will try to come up with an alternative approach in the coming days. Until then, you can already address the other remarks.

File: src/test/kialo.test.ts (hunk 4 above)
Comment (leonkamke) [reply src/test/kialo.test.ts]: Yes, I did. " import file from "./kialoTestFile.txt" " doesn't give you the content of the text file. It just gives you the file path. In this case: "/src/test/kialoTestFile.txt".

Comment (mirkolenz) [reply src/test/kialo.test.ts]: I found the following documentation entry: https://vitejs.dev/guide/assets.html#importing-asset-as-string

Does that work? Would be interesting for my upcoming changes to the testing structure.

Comment (leonkamke) [reply src/test/kialo.test.ts]: Yes, it works now.
The "?raw" suffix was missing.

Comment (mirkolenz) [reply src/test/kialo.test.ts]: Great, thank you! In principle, this PR can be merged once I have added the new way of importing the test files.

In the meantime, you could already investigate #14 and open a separate PR. This xAIF format is currently not implemented in the Python version of Arguebuf, so we later need to implement this feature in Python as well. You will have to define a new schema for this format, but if I remember correctly, the `AIF` key of xAIF directly corresponds to the "old" AIF format.

**Pull request merged**